playlists/models.py

```diff
@@ -0,0 +1,11 @@
+class Playlist:
```

Comment:
We can talk later about if this is needed. Either at all or here. You could make the argument either way, but it being in `models.py` is less than ideal.